.github/workflows/nightly.yml

```diff
@@ -32,6 +32,9 @@ on:
       windows_version:
         required: true
         type: string
+      crt_version:
```

Comment:
Nit: Can we prefix this with windows, vs, or something else?

Reply:
Updated to `vs_crt_version`